content/en/references/localstack-cli-manual.md

```diff
@@ -0,0 +1,373 @@
+---
+title: LocalStack CLI manual
+weight: 3
+description: The reference guide for LocalStack CLI commands and how to get started on using them!
+---
+
+The LocalStack command-line interface (CLI) is a tool for starting, managing, and configuring your LocalStack container.
```

Comment:
Related to @MarcelStranak 's point of missing the LocalStack CLI under the LocalStack Tools section discussed in #sig-docs [here](https://localstack-cloud.slack.com/archives/C02FZH6UB2A/p1699862643008499) (internal link):

* I agree with @MarcelStranak that under the current section structure, I'm also missing any mention of the `LocalStack CLI` under the `Tools` section, especially given that we define it as a "tool" right here. We need a landing page for the LocalStack CLI advertising why it should be used and highlighting the key features.
* I agree with @HarshCasper that the installation instructions need to be under `Getting Started`. We could use a shared block to include in `Getting Started` and `LocalStack CLI` tools intro page.

💡 One idea is to have a short advertising quick starter guide under `Tools` and then link to this `References` page for full details.